src/helpers/fileConfig.ts

```diff
@@ -23,11 +23,11 @@ export const getFileConfigHash = (options: IconJsonOptions) => {
 };
 
 const getHash = (value: string) => {
-  let hash = 0,
-    i,
-    chr;
+  let hash = 0;
+  let chr = 0;
+
   if (value.length === 0) return hash;
-  for (i = 0; i < value.length; i++) {
+  for (let i = 0; i < value.length; i++) {
```

Comment:
Move let inside the `for` loop

Example

The following `for` statement starts by declaring the variable `i` and initializing it to `0`. It checks that `i` is less than nine, performs the two succeeding statements, and increments `i` by `1` after each pass through the loop.
```javascript
for (let i = 0; i < 9; i++) {
   console.log(i);
   // more statements
}```

> Source: [Using for](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/for)

Reply:
👍